Content.Server/TimerStopwatch/TimerStopwatchComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Shared.GameObjects;
+using System;
+
+namespace Content.Server.TimerStopwatch
+{
+    //NOTE: If named "stopwatch" causes conflicts with the engine stopwatch system
+    [RegisterComponent]
+    public class TimerStopwatchComponent: Component
+    {
+        public override string Name => "TimerStopwatch";
+
+        /// <summary>
+        /// The amount of time passed since the tracking started
+        /// </summary>
+        public float PassedTime = 0;
```

Comment:
If you wanted to be able to start/stop, I'd say you should still just use game timing and just add the elapsed time to the total whenever the watch is paused, rather than incrementing the time each update.